xbmc/cores/dvdplayer/DVDCodecs/DVDFactoryCodec.cpp

```diff
@@ -216,6 +216,8 @@ CDVDVideoCodec* CDVDFactoryCodec::CreateVideoCodec(CDVDStreamInfo &hint, const C
       case AV_CODEC_ID_MPEG4:
       case AV_CODEC_ID_MSMPEG4V2:
       case AV_CODEC_ID_MSMPEG4V3:
+      case AV_CODEC_ID_MPEG1VIDEO:
+      case AV_CODEC_ID_MPEG2VIDEO:
```

Comment:
But we should follow the principle of information hiding that we already apply to the ffmpeg hw accelerators. Example:
https://github.com/xbmc/xbmc/blob/master/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.cpp#L532

Platform independent code should no be touched if something in the dependent code changes. Finally ifdefs should go away as well. IMO we should consider this next time this code is touched.